docs/guides/high-availability/deploy-aws-accelerator-fencing-lambda.adoc

```diff
@@ -2,38 +2,40 @@
 <#import "/templates/links.adoc" as links>
 
 <@tmpl.guide
-title="Deploy an AWS Lambda to guard against Split-Brain"
+title="Deploy an AWS Lambda to disable a non-responding site"
 summary="Building block for loadbalancer resilience"
 tileVisible="false" >
 
-This {section} explains how to reduce the impact when split-brain scenarios occur between two sites in a multi-site deployment.
+This {section} explains how to resolve a split-brain scenarios between two sites in a multi-site deployment.
+It also disables the replication if one site fails, to the other site can continue to serve requests.
 
 This deployment is intended to be used with the setup described in the <@links.ha id="concepts-multi-site"/> {section}.
 Use this deployment with the other building blocks outlined in the <@links.ha id="bblocks-multi-site"/> {section}.
 
 include::partials/blueprint-disclaimer.adoc[]
 
 == Architecture
-In the event of a network communication failure between the two sites in a multi-site deployment, it is no
-longer possible for the two sites to continue to replicate data between themselves and the two sites
-will become increasingly out-of-sync. As it is possible for subsequent Keycloak requests to be routed to different
-sites, this may lead to unexpected behaviour as previous updates will not have been applied to both sites.
 
-In such scenarios a quorum is commonly used to determine which sites are marked as online or offline, however as multi-site deployments only consist of two sites, this is not possible.
-Instead, we leverage "`fencing`" to ensure that when one of the sites is unable to connect to the other site, only one site remains in the loadbalancer configuration and hence only this site is able to serve subsequent users requests.
+In the event of a network communication failure between the two sites in a multi-site deployment, it is no longer possible for the two sites to continue to replicate data between them.
+As this setup values consistency over availability, and as the {jdgserver_name} caches are configured with a failure policy `FAIL`, all user requests are served with an error message until this is resolved by either restoring the network connection or disabling the replication.
```

Comment:
I would consider rewording this slightly, mabye:

"The {jdgserver_name} is configured with a `FAIL` failure policy, which ensures consistency over availability. Consequently all user requests are served with an error message until the failure is resolved, either by restoring the network connection or by disabling cross-site replication."